### ISSUE TITLE
gradients of noisy loss w.r.t parameter \theta

### ISSUE THREAD

**Issue by qingerVT** [uber-research/learning-to-reweight-examples#2]
The basic procedure sounds like 
- (a) set \epsilon = 0, get grads to update \theta
- (b) evaluate new \theta on clean validation set . 
- (c) set - grads (validation loss w.r.t \epsilon) as \epsilon
- (d) use new \epsilon to evaluate noisy data and update parameters again

My question is when \epsilon=0, the derivative of loss w.r.t. \theta =0? That means we don't update \theta actually in (a)?

**Comment by qingerVT** [comment]
Sorry, got it!

**Comment by youkaichao** [comment]
How do you figure it out? I have the same problem.

**Comment by haoshuang0223** [comment]
@qingerVT So what is the answer? I also get confused about it recently.

**Comment by iTomxy** [comment]
Same confusion. Can you explain a bit about it? @qingerVT
